internal/pkg/jsonnet/context_test.go

```diff
@@ -113,3 +114,84 @@ func TestVmContext_VariablesTypes(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, strings.TrimLeft(expected, "\n"), output)
 }
+
+func TestVmContext_Notifier(t *testing.T) {
+	t.Parallel()
+	notifier := &testNotifier{}
+	ctx := NewContext()
```

Comment:
Rovnaky test, aky je v `go-jsonnet` libke, len tu sa testuje, ze to funguje spolu s `jsonnet.Context`.